src/actions/genSale/purchaseDomains.ts

```diff
@@ -0,0 +1,121 @@
+import * as ethers from "ethers";
+import { getSaleStatus } from ".";
+import { GenSale } from "../../contracts/types";
+import { Claim, SaleStatus, Mintlist, Maybe, GenSaleStatus } from "../../types";
```

Comment:
Remove `SaleStatus` as it is no longer used

Reply:
removed